lib/string-utils.js

```diff
@@ -0,0 +1,10 @@
+const getLink = str => {
+  const containsLink = str.match(/\(https?:(.*)\)/);
```

Comment:
This seems like it would be more readable with a named group...something like

`\((?<url>https?:.*)\)`

and then it could be `return containsLink.groups.url`

So that there doesn't need to be any replacing of the "( )"s.

Reply:
Much better. Many thanks. Done